meson_options.txt

```
@@ -437,6 +437,16 @@ option('systemd_root_prefix',
   value: '',
   description: 'Directory to base systemd’s installation directories on',
 )
+option('launchd',
+  type: 'feature',
+  value: 'auto',
+  description: 'Launchd auto-launch support'
```

As this is mostly a Linux project; how about
```suggestion
  description: 'Launchd auto-launch support (MacOS only)'
```